packages/schemas/src/player/gamemodes/duels/index.ts

```diff
@@ -3,6 +3,15 @@ import { Field } from '../../../metadata';
 import { BridgeDuels, MultiDuelsGameMode, SingleDuelsGameMode, UHCDuels } from './mode';
 
 export class Duels {
+  @Field()
+  public pingRange: number;
```

Comment:
These don't get initiated anywhere. Also add { store: { default: 300 } } to the pingRange decorator.